lib/beaker/host/windows/exec.rb

```diff
@@ -119,6 +119,30 @@ def selinux_enabled?()
     false
   end
 
+  # Create the provided directory structure on the host
+  # @param [String] dir The directory structure to create on the host
+  # @return [Boolean] True, if directory construction succeeded, otherwise False
+  def mkdir_p dir
+    cmd = "mkdir -p \"#{dir}\""
+    result = exec(Beaker::Command.new(cmd), :acceptable_exit_codes => [0, 1])
+    result.exit_code == 0
+  end
+
+  # Recursively remove the path provided
+  # @param [String] path The path to remove
+  def rm_rf path
+    execute("rm -rf #{path}")
```

Comment:
Perhaps this should be double-quoted to prevent globbing, as well?

Reply:
This is a particular case, we have some tests that use `rm -rf /path/*` which does not delete anything if quoted or double quoted.